src/editor/components/modals/file-modal/manage-file-sources/validators.js

```diff
@@ -0,0 +1,39 @@
+const ALLOWED_PROTOCOLS = ["https://", "http://"];
```

Comment:
I'm not sure if this file should go into some other shared dir, but I think for now it should be as closely colocated to the code that uses these functions.